proxies/ruby/helpers.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+def get_entity_from_location(location, data_store)
+  url_parts = location.sub(%r{^/}, '').split('/')
+  location_type, *location_path = url_parts
+  case location_type
+  when 'command'
+    entity_type, entity_id = location_path
+    return data_store[entity_type.to_sym][entity_id]
+  when 'client'
+    client_id = location_path[0]
+    return data_store[:clients][client_id]
+  end
+  nil
+end
+
+def parse_params(body, params)
+  parsed_params = []
+  params.each do |param|
+    if param['callbackURL']
+      parsed_params.push(param['callbackURL'])
+    elsif param['type']
+      parsed_params.push(get_entity_from_param_type(param['type'], body))
+    else
+      parsed_params.push(param['value'])
+    end
+  end
+  parsed_params
+end
+
+def get_entity_from_param_type(type, body)
+  case type
+  when 'user'
+    req_body_user = body['user']
+    user_data = DevCycle::UserData.new
+    user_data.user_id = req_body_user['user_id'] if req_body_user['user_id']
+    user_data.email = req_body_user['email'] if req_body_user['email']
+    user_data.name = req_body_user['name'] if req_body_user['name']
+    user_data.language = req_body_user['language'] if req_body_user['language']
+    user_data.country = req_body_user['country'] if req_body_user['country']
+    user_data.app_version = req_body_user['appVersion'] if req_body_user['appVersion']
```

Comment:
are these not camel cased?
https://github.com/DevCycleHQ/ruby-server-sdk/blob/main/lib/devcycle-ruby-server-sdk/models/user_data.rb#L39

Reply:
yep, you're right. Will fix